WooCommerce/Classes/ViewRelated/Orders/Order Details/Review Order/ReviewOrderViewModel.swift

```diff
@@ -3,9 +3,27 @@ import Yosemite
 import protocol Storage.StorageManagerType
 
 final class ReviewOrderViewModel {
+    /// Quick access to header types for table view registration
+    ///
+    let allHeaderTypes: [UITableViewHeaderFooterView.Type] = {
+        [PrimarySectionHeaderView.self,
+         TwoColumnSectionHeaderView.self]
+    }()
```

Comment:
nit: To avoid having to go back and forth between the VC and VM to see how `allHeaderTypes` and `allCellTypes` are defined and used, how about calling a method that defines and registers them together? (For example, [here's how that's done in OrderDetailsViewModel](https://github.com/woocommerce/woocommerce-ios/blob/c6a1a399eea1cae2035264d46037d6db5e361f2a/WooCommerce/Classes/ViewModels/Order%20Details/OrderDetailsViewModel.swift#L197-L208).)